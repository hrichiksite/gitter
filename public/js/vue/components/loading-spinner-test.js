const mount = require('../__test__/vuex-mount');
const { default: LoadingSpinner } = require('./loading-spinner.vue');

describe('loading-spinner', () => {
  it('matches snapshot', () => {
    const { wrapper } = mount(LoadingSpinner);
    expect(wrapper.element).toMatchSnapshot();
  });
});
