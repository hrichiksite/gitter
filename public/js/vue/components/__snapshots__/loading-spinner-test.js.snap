// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading-spinner matches snapshot 1`] = `
<span
  class="loading-spinner"
>
  <span
    class="loading-spinner-label"
  >
    Loading
  </span>
</span>
`;
